docker-compose.manager.yml

```
@@ -0,0 +1,55 @@
+services:
+  admin-dashboard:
+    image: ragapp/admin-dashboard
```

how about just naming this manager?
```suggestion
    image: ragapp/manager
```